### PR TITLE
Saving an activity report feedback

### PR DESCRIPTION
**Pull request by jasalisbury**
**Description of change**
 * Rename "activityParticipant" to "activityRecipient"
 * Remove `--runInBand`, it appears to no longer be needed to prevent a race with tests that hit the database

**How to test**
Pull down, run all tests

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/98

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "server:debug": "nodemon src/index.js --exec babel-node --inspect",
     "client": "yarn --cwd frontend start",
     "test": "jest src",
-    "test:ci": "cross-env JEST_JUNIT_OUTPUT_DIR=reports JEST_JUNIT_OUTPUT_NAME=unit.xml POSTGRES_USERNAME=postgres POSTGRES_DB=ttasmarthub CURRENT_USER_ID=5 CI=true jest src tools --runInBand --coverage --reporters=default --reporters=jest-junit",
+    "test:ci": "cross-env JEST_JUNIT_OUTPUT_DIR=reports JEST_JUNIT_OUTPUT_NAME=unit.xml POSTGRES_USERNAME=postgres POSTGRES_DB=ttasmarthub CURRENT_USER_ID=5 CI=true jest src tools --coverage --reporters=default --reporters=jest-junit",
```
Comment (rahearn) [inline package.json:21]: 💯

**Review by rahearn — CHANGES_REQUESTED** [review]
This is a lot cleaner, thanks.

Before merging, the newly renamed `activity-recipients.yaml` doc still references the old participant schema. Also, would it be possible to add a quick descriptive sentence at least to each overall object in the API docs, if not each field as well, especially any that aren't obvious.

**Comment by rahearn** [comment]
It looks like the `paticipant` reference is still there, which prevents redoc from working: https://github.com/adhocteam/Head-Start-TTADP/blob/664ebc1ec918b9705b9e3a8d512283e5ec373467/docs/openapi/paths/activity-reports/activity-recipients.yaml#L19

**Comment by jasalisbury** [comment]
> It looks like the `paticipant` reference is still there, which prevents redoc from working:
> 
> https://github.com/adhocteam/Head-Start-TTADP/blob/664ebc1ec918b9705b9e3a8d512283e5ec373467/docs/openapi/paths/activity-reports/activity-recipients.yaml#L19

Oh forgot to update that whole file! Should be updated now. I had to hard refresh locally to get redoc to show the new changes

**Review by rahearn — APPROVED** [review]
🚀

**Pull request merged**